### PR TITLE
feat: use tailwind and style

### PR DESCRIPTION
**Pull request by portenez**
Signed-off-by: vgarciavalen <info@vgarcia.dev>

*Issue number of the reported bug or feature request: #<number>*

**Describe your changes**
- Remove styled components
- Setup tailwind
- style 

**Testing performed**
- Jest passing
- Manual test using GCP

**Additional context**
- Switched to tailwind to iterate faster

### CODE REVIEW

File: src/__tests__/__snapshots__/Home.spec.tsx.snap [hunk 1]
```diff
@@ -2,17 +2,21 @@
...
-  <h1>
-    Welcome to the homepage, 
-    pqrs
-  </h1>
+  <div
+    class="p-10"
+  >
+    <h2>
+      Welcome to the homepage, 
+      pqrs
```
Comment (dakotalillie) [inline src/__tests__/__snapshots__/Home.spec.tsx.snap]: **Question**

What is pqrs?

File: src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default React.forwardRef<
+  HTMLButtonElement,
+  React.HTMLProps<HTMLButtonElement>
+>(function Button({ onClick, children }, reference) {
```
Comment (dakotalillie) [inline src/components/Button.tsx]: **Nitpick**

Should we past `...rest` props on to the underlying button element?

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 
 const HomePage: NextPage = () => (
-  <>
-    <h1>Welcome to Coffee Chat</h1>
-    <p>This is work in progress</p>
-    <Link href="/login/oidc" passHref>
-      <button type="button">log in here</button>
-    </Link>
-  </>
+  <div className="container mx-auto  mt-11  px-4 pt-4 pb-10 space-y-3 w-80 bg-slate-100 text-center">
```
Comment (dakotalillie) [inline src/pages/index.tsx]: **Nitpick**

```suggestion
  <div className="container mx-auto mt-11 px-4 pt-4 pb-10 space-y-3 w-80 bg-slate-100 text-center">
```

**Review by dakotalillie — DISMISSED** [review]

**Review by Abhilasha3112 — DISMISSED** [review]

**Comment by portenez** [comment]
@Abhilasha3112 and @dakotalillie I've addressed the PR comments and squashed merged. Can you all add your reviews again?

File: jest.config.js [hunk 4]
```diff
@@ -1,11 +1,18 @@
...
   testMatch: ['**/__tests__/**/*.spec.[jt]s?(x)'],
   setupFiles: ['<rootDir>/.jest/storyBookTesting.ts'],
   setupFilesAfterEnv: ['<rootDir>/.jest/jestDomSetup.ts'],
-  testEnvironment: 'jsdom',
+  testEnvironment: 'jest-environment-jsdom',
   globalSetup: '<rootDir>/.jest/globalSetup.ts',
-  transform: {
-    '\\.(js|jsx|tsx|ts)$': ['babel-jest', { configFile: './jest.babelrc' }],
+  moduleNameMapper: {
+    '\\.(scss|sass|css)$': 'identity-obj-proxy',
```
Comment (Abhilasha3112) [inline jest.config.js:16]: what does identity-obj-proxy do? i read it's doc but it wasn't clear. can you give an example?

File: src/pages/_app.tsx [hunk 5]
```diff
@@ -1,19 +1,45 @@
...
-  >
-    <GlobalStyle />
-    <Component {...pageProps} />
-  </SWRConfig>
-)
+const AppBar: FC = () => {
+  const { session } = useContext(UserContext)
+  return (
+    <div className="flex flex-row py-4 px-8 items-baseline justify-between">
+      <div className="text-3xl font-display font-extrabold">Coffee Chat</div>
```
Comment (Abhilasha3112) [inline src/pages/_app.tsx:12]: should we start using https://www.npmjs.com/package/clsx ? 
we found it pretty useful in our latest tailwind project.

**Review by Abhilasha3112 — APPROVED** [review]
lgtm

File: src/pages/_app.tsx (hunk 5 above)
Comment (portenez) [reply src/pages/_app.tsx:12]: Looked at it. I don't think we need it yet. If we do, we can bring it in. But not yet given the current usage.

File: jest.config.js (hunk 4 above)
Comment (portenez) [reply jest.config.js:16]: it's pretty much a mock. that returns same in same out.

**Pull request merged**